public/static/js/subscriptions.js

```diff
@@ -33,17 +56,17 @@ function subscriberLabel(subscriber) {
 // }
 
 // Return a promise for the add subscription modal, with the list of triggers
-function addSubscriber($mdDialog, triggers) {
-    return subscriberPromise($mdDialog, "Add", triggers)
+function addSubscriber($mdDialog, triggers, omitMethods) {
+    return subscriberPromise($mdDialog, "Add", triggers, undefined, omitMethods)
 }
 
 // Return a promise for the edit subscription modal, with the list of triggers.
 // trigger and subscriber are the selected trigger and subscriber
-function editSubscriber($mdDialog, triggers, subscription) {
-    return subscriberPromise($mdDialog, "Edit", triggers, subscription)
+function editSubscriber($mdDialog, triggers, subscription, omitMethods) {
+    return subscriberPromise($mdDialog, "Edit", triggers, subscription, omitMethods)
 }
 
-function subscriberPromise($mdDialog, verb, triggers, subscription) {
+function subscriberPromise($mdDialog, verb, triggers, subscription, omitMethods) {
```

Comment:
How about reversing the order of subscription and omitMethods., and adding if subscription === undefined; subscription = {}?

Reply:
done